src/main/java/net/troja/eve/esi/HeaderUtil.java

```diff
@@ -0,0 +1,82 @@
+package net.troja.eve.esi;
+
+import com.fasterxml.jackson.databind.util.ISO8601DateFormat;
+import java.text.ParseException;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class HeaderUtil {
+
+    private static final ISO8601DateFormat DATE_FORMAT = new ISO8601DateFormat();
```

Comment:
This one is thread-safe, right?

Reply:
It say it is in documentation
> Provide a fast thread-safe formatter/parser DateFormat for ISO8601 dates ONLY.
It was mainly done to be used with Jackson JSON Processor.